css/style.css

```diff
@@ -289,4 +289,64 @@ footer p {
 }
 
```

Comment:
I'd recommend changing `#benefits p, #benefits ul` to be `max-width: 600px;` otherwise it will overflow when you shrink down your browser